test-alien-bind.js

```diff
@@ -0,0 +1,85 @@
+/* global describe, it, expect, beforeEach, afterEach */
+var sinon = require('sinon')
+
+describe('bind', function () {
+  var bind
+
+  beforeEach(function () {
+    sinon.spy(Function.prototype, 'bind')
+    bind = require('./alien-bind')
```

Comment:
In underscore, the native bind is set as _.bind in the require so this would catch that. Not sure if it executes every time though.